app/menus/menus/window.js

```diff
@@ -1,15 +1,14 @@
-const {accelerators} = require('../../accelerators');
-
-module.exports = function () {
+module.exports = function (commands) {
   return {
     role: 'window',
     submenu: [
       {
         role: 'minimize',
-        accelerator: accelerators.minimize
+        accelerator: commands['window:minimize']
       },
       {
-        role: 'zoom'
+        role: 'zoom',
```

Comment:
Given that it does nothing on Windows (and KDE), should it even be in the menus on Windows (or Linux)?